### ISSUE TITLE
[MHV SM / Accessibility] Landing Page: H2 - Broken Aria reference

### ISSUE THREAD

**Issue by steveg-IAT** [department-of-veterans-affairs/va.gov-team#119858]
### Point of contact

@steveg-IAT

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

WCAG 1.3.1: Info and Relationships (Level A): Broken ARIA reference

https://staging.va.gov/my-health/secure-messages/inbox: 
Initial page state > "Filter Messages in inbox" H2 has aria-describedby="filter-clear-success" but is missing the referenced target. This error goes away once a filter is applied and then cleared.

### Reproduction steps

1. With [Stage User #7](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/sm/test-accounts.md)
2. Navigate to the MHV: Messages landing page

The issue presents itself via code review.
...

### Proposed solution or next steps

...

### References, articles, or WCAG support

1. [WCAG 1.3.1: Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)
2.
3.
...

Screenshot:
<img width="1269" height="810" alt="Image" src="https://github.com/user-attachments/assets/ff6a87e5-3dfa-4d4b-8130-2b8966ae9546" />

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Voice command
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [x] Semantic HTML
- [x] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by steveg-IAT** [comment]
Using VoiceOver (VO), the aria-describedby works correctly when a filter is applied and the Clear Filter button is clicked, which returns the focus to the H2, and correctly reads: 'filter message in inbox - filters **successfully** cleared'. 

(*in the generated span#filter-clear-success element 'successfully' is also mispelled and causes the screen reader (VO) to mispronounce the word... /mhv-secure-messaging/components/Search/SearchForm.jsx :: Line: 358)

I feel this H2 should have three states for the aria if we're going to apply one:
- 'filter message in inbox - no filters applied' -> default
- 'filter message in inbox - filters successfully cleared' -> post cleared (current functionality)
- 'filter message in inbox - filters applied' -> filters applied and user returns to the heading (this could be more detailed if we feel it's needed)

**Comment by sofia-cordero** [comment]
Hey @steveg-IAT while working on this ticket, I noticed that there was no error state on this component's `aria-describedBy` for when the filters are not applied successfully.  Is that something we'd want to add in a follow up ticket?
